#request
float64[] target_points
---
#response
bool success